src/Erl/Data/Binary.erl

```diff
@@ -1,12 +1,14 @@
 -module(erl_data_binary@foreign).
--export([byteSize/1, part_/5]).
+-export([byteSize/1, part/3]).
+
+-define(nothing, {nothing}).
+-define(just(A), {just, A}).
 
 byteSize(X) -> erlang:byte_size(X).
 
-part_(Left, Right, Start, Len, Binary) ->
+part(Start, Len, Binary) ->
```

Comment:
I'd stick with the standard original style passing in `Left` and `Right` - though I agree with the change to use `Nothing`/`Just`